packages/testing/playwright/tests/e2e/chat-hub/chat-hub-basic.spec.ts

```diff
@@ -0,0 +1,151 @@
+import { INSTANCE_OWNER_CREDENTIALS } from '../../../config/test-users';
+import { test, expect } from '../../../fixtures/base';
+import { ChatHubChatPage } from '../../../pages/ChatHubChatPage';
+import { CredentialModal } from '../../../pages/components/CredentialModal';
+
+const mockAnthropicApiKey = 'mock-anthropic-api-key';
+const anthropicApiKey =
+	process.env.ANTHROPIC_API_KEY /* for recording requests while development */ ??
```

Comment:
While development `ANTHROPIC_API_KEY=<real api key> pnpm test:container:standard:ui --grep chat-hub --ui` to get mock JSON files populated under expectations/chat-hub.